plugins/occt/module/Testing/TestF3DOCCTReader.cxx

```diff
@@ -8,14 +10,23 @@
 
 int TestF3DOCCTReader(int vtkNotUsed(argc), char* argv[])
 {
-  std::string filename = std::string(argv[1]) + "data/f3d.stp";
-  vtkNew<vtkF3DOCCTReader> reader;
-  reader->RelativeDeflectionOn();
-  reader->SetLinearDeflection(0.1);
-  reader->SetAngularDeflection(0.5);
-  reader->ReadWireOn();
-  reader->SetFileName(filename);
-  reader->Update();
-  reader->Print(cout);
-  return reader->GetOutput()->GetNumberOfBlocks() > 0 ? EXIT_SUCCESS : EXIT_FAILURE;
+  for (const auto file : {
+         "data/f3d.stp",
+         "data/f3d.igs",
+         "data/f3d.brep",
+       })
+  {
+    std::string filename = std::string(argv[1]) + file;
+    vtkNew<vtkF3DOCCTReader> reader;
+    reader->RelativeDeflectionOn();
+    reader->SetLinearDeflection(0.1);
+    reader->SetAngularDeflection(0.5);
```

Comment:
you would need a `reader->SetFileFormat(vtkF3DOCCTReader::FILE_FORMAT::BREP);`  and same for iges.